lib/game/base_game.dart

```diff
@@ -139,7 +137,7 @@ class BaseGame extends Game with FPSCounter {
   @override
   @mustCallSuper
   void onResize(Vector2 size) {
-    this.size = size;
+    this.size.setFrom(size);
```

Comment:
the perks of mutability

Reply:
Yeah, I guess we should settle for what we want, it can be both a perk and a confusion I think.